src/containers/quiz/quiz-header/QuizHeader.tsx

```diff
@@ -22,6 +28,13 @@ type QuizHeaderProps = {
   questionsAnswered: number
   createdAt: string
   updatedAt: string
+  isNotStarted: boolean
+  quizItems: Question[]
+  settings: {
+    attemptLimit: QuizAttempt
+    timeLimit: QuizTimeLimit
+  }
```

Comment:
```suggestion
  attemptLimit: QuizAttempt
  timeLimit: QuizTimeLimit
```
I think it is better to not use nesting in this case, but it's up to you.

Reply:
I think I should leave as it was